app/images/icons/bridge.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
+  <path d="m74 284c-21 7-36 26-36 49 0 28 23 51 52 51 28 0 51-23 51-51 0-20-12-38-28-46 14-65 73-114 143-114 47 0 89 22 116 56 12-6 25-10 39-11-33-50-90-84-155-84-90 0-165 65-182 150z m400 49c0 28-23 51-52 51-28 0-51-23-51-51 0-28 23-51 51-51 29 0 52 23 52 51z"/>
+</svg>
```

Comment:
Nice work! not sure if you need a line break here after the closing tag but GitHub is complaining.

<img width="472" alt="Screenshot 2023-03-01 at 3 33 58 PM" src="https://user-images.githubusercontent.com/8112138/222291026-75de75c3-5f9e-46f3-b0df-c39337a65555.png">

Reply:
Thanks! I just added a newline to the end to address this